### DISCUSSION TITLE
Error when trying to connect

### DISCUSSION THREAD

**Discussion by charlotattardjpj** [drakkan/sftpgo#477] [General]
HI, 

Im experincing an issue when trying to connect with every user on the SFTP with the below error code

Operation timed out (Connection timed out). The connection attempt was rejected. The server may be down, or your network may not be properly configured.

Apart from that the logs are set to info, is there a possibility I could set them to Debug after the software has been installed ?

Thanks,

**Comment by drakkan** [comment]
Hi,

yes please take a look [here](https://github.com/drakkan/sftpgo/blob/main/docs/full-configuration.md). `--log-verbose` is true by default. If you have connection time out and you see non logs this means that you have a firewall or something other that blocks the traffic
